src/carter-mc-barter-narter-is-dope.css

```diff
@@ -1,7 +1,32 @@
-body {
-  min-height: 100vh;
+div{
```

Comment:
When we're thinking about CSS (and codebases generally), we want to think about how our system will _scale over time_. With this change, we're saying: "All `div`s should have a min-height of 5% of the viewport, and content should be flex-centered inside it both vertically and horizontally".

I'm guessing that's a bit more heavy-handed than we want! Instead, we can make a best guess in how these styles might be used in the future, make them more modular, and represent them as CSS _classes_ so they're more easily composable for future explorers.

For example, we could make our flex stuff
```
.flex-centered {
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: center;
}
```
... since it's likely we'll want to create other UI components with vertically-and-horizontally-centered contents.

For the min-height, I'd hesitate a guess that we won't me reusing that all over the place - so for now, we can treat this as more of a one-off with a descriptive classname:
```
.title-container {
  min-height: 5vh;
}
```
then, on our div on the page, we can use
```
<div class="title-container flex-centered">
```
A bit of a "pro tip": a common issue that recurs in CSS over time are early, too-prescriptive styles for basic html elements (`div`, `a`, etc) needing to be overridden constantly. A way to avoid this is using small, modular utility classes (`flex-centered`) and specific names for individual UI elements (`title-container`)

There's a lot further we can take this - like BEM, or "Block - Element - Modifier" - to avoid the dreaded _CSS selector specificity battles_, but we should be well-served by this for now... and avoid another pitfall in software engineering of "over engineering", or building too many assumptions into current code for how we think the codebase will grow in the future. 

Knowing where to draw the line on over-engineering is both art and science. Let's continue discussing this concept as we proceed down the silicon-paved path of ✨ software engineering ✨